### PR TITLE
add `emby-server-deb`

### PR DESCRIPTION
**Pull request by warofzen**
Add a pacscript to install [Emby Server](https://github.com/MediaBrowser/Emby.Releases) from their .deb release

### CODE REVIEW

File: packages/emby-server-deb/emby-server-deb.pacscript [hunk 1]
```diff
@@ -0,0 +1,8 @@
+name="emby-server-deb"
+version="4.6.7.0"
+gives="emby-server"
+breaks="${gives} ${gives}-bin ${gives}-app"
```
Comment (wizard-28) [inline packages/emby-server-deb/emby-server-deb.pacscript:4]:
```suggestion
breaks="${gives} ${gives}-bin ${gives}-app ${gives}-git"
```

**Review by wizard-28 — CHANGES_REQUESTED** [review]
@Henryws I think it's okay to not contact the devs for this.

**Review by wizard-28 — APPROVED** [review]

**Comment by wizard-28** [comment]
@warofzen the CI fails in all OSes

**Comment by warofzen** [comment]
It seems that the CI cannot create a symlink for the `emby-server.service` file, but it worked just fine on my local machine:
https://asciinema.org/a/DwwakO0Zdpf6LiibqCtP1WniB

**Comment by wizard-28** [comment]
> It seems that the CI cannot create a symlink for the `emby-server.service` file, but it worked just fine on my local machine: https://asciinema.org/a/DwwakO0Zdpf6LiibqCtP1WniB

Okay, so @Henryws will test it on his machine, and then we can merge.

**Pull request merged**